src/gps/gps_util.h

```diff
@@ -1,6 +1,6 @@
 #pragma once
 
-#include "../navtypes.h"
+#include "../../src/navtypes.h"
```

Comment:
This change isn't necessary, they're the same paths. Could you remove this change?